pkg/kubecost/totals.go

```diff
@@ -108,9 +123,12 @@ func ComputeAllocationTotals(as *AllocationSet, prop string) map[string]*Allocat
 		arts[key].CPUCostAdjustment += alloc.CPUCostAdjustment
 		arts[key].GPUCost += alloc.GPUCost
 		arts[key].GPUCostAdjustment += alloc.GPUCostAdjustment
-		arts[key].LoadBalancerCost += alloc.LBTotalCost()
-		arts[key].NetworkCost += alloc.NetworkTotalCost()
+		arts[key].LoadBalancerCost += alloc.LoadBalancerCost
+		arts[key].LoadBalancerCostAdjustment += alloc.LoadBalancerCostAdjustment
+		arts[key].NetworkCost += alloc.NetworkCost
+		arts[key].NetworkCostAdjustment += alloc.NetworkCostAdjustment
 		arts[key].PersistentVolumeCost += alloc.PVCost()
```

Comment:
Hm, again not blocking but it feels a little inconsistent to use PVCost() here-- is the difference here that no adjustment is taken into account on that one? If this is working though, no need to block.

Reply:
Correct. Kind of rough naming conventions, but the counterpart that includes adjustments is `PVTotalCost()`. These adjustments cause some major readability/naming issues, at least as-written.